src/Symfony/Component/Routing/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -11,33 +11,46 @@
 
 namespace Symfony\Component\Routing\Tests\Loader;
 
+use InvalidArgumentException;
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Config\FileLocator;
-use Symfony\Component\Routing\Loader\YamlFileLoader;
 use Symfony\Component\Config\Resource\FileResource;
+use Symfony\Component\Routing\Loader\YamlFileLoader;
```

Comment:
to be reverted: even ordering should be preserved

Reply:
It's so hard not to use my IDE...